doc/GithubPRs.md

```diff
@@ -0,0 +1,37 @@
+GitHub PRs workflow
+======
+
+See also https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
```

Comment:
I'd rather we used the devguide version of this: https://suricata.readthedocs.io/en/latest/devguide/codebase/contributing/contribution-process.html
:)

Reply:
Indeed, I did not know this part